lora/lora.py

```diff
@@ -138,8 +138,22 @@ def __len__(self):
 
 
 def load(args):
-    names = ("train", "valid", "test")
-    train, valid, test = (Dataset(Path(args.data) / f"{n}.jsonl") for n in names)
+    try:
+        train = Dataset(Path(args.data) / "train.jsonl")
+    except Exception as e:
+        print(f"Training set not built at {args.data}/train.jsonl ({e})")
+        raise
+    try:
+        valid = Dataset(Path(args.data) / "valid.jsonl")
+    except Exception as e:
+        print(f"Validation set not built at {args.data}/valid.jsonl ({e})")
+        raise
+    try:
+        test = Dataset(Path(args.data) / "test.jsonl")
+    except Exception as e:
+        print(f"Test set not built at {args.data}/test.jsonl ({e})")
+        raise
```

Comment:
Maybe cleaner to do this in a loop:
```suggestion
   def load_and_check(name):
      dataset_path = Path(args.data) / f"{name}.jsonl"
      try:
          train = Dataset(dataset_path)
      except Exception as e:
          print(f"Unable to build dataset {dataset_path} ({e})")
          raise
     train, valid, test = (load_and_check(n) for n in names)
```